### PR TITLE
feat(integration): integrate backend routes with frontend with pagination and filters

### PR DESCRIPTION
**Pull request by senthil2000-dev**

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -27,6 +28,10 @@
 app.include_router(member_router, tags=["auth"], prefix="/auth")
 app.include_router(bot_router, tags=["bot"], prefix="/bot")
 
+# app.include_router(member_router, tags=["Auth"], prefix="/auth")
+# app.include_router(bot_router, tags=["Bot"], prefix="/bot")
```
Comment (bestinbthomas) [inline backend/main.py:32]: why are they commented out? @kelpikz

Comment (kelpikz) [reply backend/main.py]: These routes have already been included in the previous line, ig it can be removed

Comment (kelpikz) [reply backend/main.py]: These routes have already been included in the previous line, ig it can be removed

File: frontend/src/js/data/scrums.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+const endDateId = 'end_date_id' 
+
+function handleApplyDateRange(data){
+  let startDate = data.detail.startDate.split('-').reverse().join('-');
+  let endDate = data.detail.endDate.split('-').reverse().join('-');
+  window.location.href = `/scrum/${startDate}/${endDate}`;
+}
+
+onMount(async () => {
+  scrumMaster = "Bestin B Thomas";
```
Comment (bestinbthomas) [inline frontend/src/js/data/scrums.js:37]: can you move this to config?

**Review by bestinbthomas — CHANGES_REQUESTED** [review]

**Review by bestinbthomas — APPROVED** [review]

**Pull request merged**